core/src/main/java/io/grpc/internal/AbstractClientStream.java

```diff
@@ -102,11 +101,11 @@ void writeFrame(
   protected AbstractClientStream(
       WritableBufferAllocator bufferAllocator,
       StatsTraceContext statsTraceCtx,
-      @Nullable TransportTracer transportTracer,
+      TransportTracer transportTracer,
       Metadata headers,
       boolean useGet) {
     Preconditions.checkNotNull(headers, "headers");
-    this.transportTracer = transportTracer;
+    this.transportTracer = Preconditions.checkNotNull(transportTracer, "transportTracer");
```

Comment:
optional: statically importing this makes the code shorter usually.

Reply:
done